### ISSUE TITLE
New feature (if possible)

### ISSUE THREAD

**Issue by InfanteD** [xAranaktu/FIFA-23-CM-Cheat-Table#29]
Is it possible to add the contrac modifications like the FIFA22 cheat table? sorry, i'm just asking, not demanding

**Comment by MinisterFasola** [comment]
> Is it possible to add the contrac modifications like the FIFA22 cheat table? sorry, i'm just asking, not demanding

yeah agree. it will be pleased to have contract modification features
